notify/notify.go

```diff
@@ -293,7 +296,7 @@ func NewMetrics(r prometheus.Registerer) *Metrics {
 		"telegram",
 	} {
 		m.numNotifications.WithLabelValues(integration)
-		m.numTotalFailedNotifications.WithLabelValues(integration)
+		m.numTotalFailedNotifications.WithLabelValues(integration, "")
```

Comment:
This does not make a lot of sense... I think we want to populate here counters with each possible 3 status codes: `4xx`, `5xx` and unknown.